chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -890,5 +890,14 @@
             <recover>false</recover>
         </job>
         
+        <job>
```

Comment:
One more thing - the PR shows a conflict in this file. Requesting changes to address that.